server/svix-server/src/v1/utils/mod.rs

```diff
@@ -462,6 +465,42 @@ pub fn get_unix_timestamp() -> u64 {
         .as_secs()
 }
 
+#[derive(Deserialize, JsonSchema)]
+pub struct ApplicationEndpointPath {
+    pub app_id: ApplicationIdOrUid,
+    pub endpoint_id: EndpointIdOrUid,
+}
+
+#[derive(Deserialize, JsonSchema)]
+pub struct ApplicationMsgPath {
```

Comment:
I'm on the fence whether this should be `Message` or kept `Msg`. I think we call it `msg` everywhere, like really everywhere, so it's probably fine. Though it does stick out a bit. :)

Reply:
Yep, I went with `Msg` because it's so widely used, will stick to that.